requirements.txt

```diff
@@ -121,7 +121,7 @@ typing==3.6.4 # needed for django-extensions
 django-stagingcontext==0.1.0
 django-ga-context==0.1.0
 django-impersonate==1.3
-django-filter==1.1.0
+django-filter==1.1.0 # pyup: <1.2.0
```

Comment:
we can probably pin it to <2.0.0, since if they make another 1.x release it's more likely to be compatible.